docs/080-breaking-changes.rst

```diff
@@ -46,6 +46,26 @@ New Restrictions
   3. Explicit conversions between literals and enums are only allowed if the literal can
      represent a value in the enum.
 
+* There are new restrictions on explicit conversions between two different types. The conversion is
+  only allowed when there is at most one change in sign, width or 'kind' / different type (``int``,
+  ``address``, ``bytesNN``, etc.) For example, the conversion ``uint16(int8)`` is disallowed since
+  the conversion changes width (8 bits to 16 bits) and sign (signed integer to unsigned integer.) To
+  get the previous behaviour, use an intermediate conversion. In the previous example, this would be
+  ``uint16(uint8(int8))`` or ``uint16(int16(int8))``. The following are some examples of conversions
+  that are disallowed by this rule. Note that, given types ``T`` and ``S``, the notation ``T(S)``
+  refers to the explicit conversion ``T(x)``, where ``x`` is any arbitrary variable of type ``S``.
+
+  - ``address(uint)`` and ``uint(address)``: converting both 'kind' and width. Replace this by
```

Comment:
I still find this `'kind'` weird, especially because it's always single quoted

Reply:
A problem with using type would be: distinguishing the difference between `uint16` v/s `int8` and 'uint16` v/s `address`. Each pair has different type, but the latter one has different kind. 

But what do you think is a good replacement?

Reply:
type category?

Reply:
Agree with type category

Reply:
Replaced 'kind' by type-category.